src/mixtures/mixturemodel.jl

```diff
@@ -500,6 +500,27 @@ componentwise_logpdf(d::MultivariateMixture, x::AbstractVector) = componentwise_
 componentwise_logpdf(d::MultivariateMixture, x::AbstractMatrix) = componentwise_logpdf!(Matrix{eltype(x)}(undef, size(x,2), ncomponents(d)), d, x)
 
 
+function quantile(d::UnivariateMixture{Continuous}, p::Real)
+    ps = probs(d)
+    min_q, max_q = extrema(quantile(component(d, i), p) for (i, pi) in enumerate(ps) if pi > 0)
+
+    if iszero(p)
+        return(min_q)
+    elseif isone(p)
+        return(max_q)
+    end
+
+    tol =  Base.rtoldefault(typeof(p)) * abs(p)
+
+    if abs(cdf(d, min_q) - p) < tol
+        return(min_q)
+    elseif abs(cdf(d, max_q) - p) < tol
+        return(max_q)
+    end
```

Comment:
I guess it would be reasonable to handle this in the general algorithm in https://github.com/JuliaStats/Distributions.jl/blob/d6de7214e6d84df77d1eb3ec7a98d23862d1232d/src/quantilealgs.jl#L3-L22.